scripts/cluster/yarn_cluster_setup/move_files_to_master.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# current_path=`pwd`
+# library_file_address=$current_path/bash_scripts_library/include_functions.bash
+# source $library_file_address
+
+# get_nodes_info
+# authorize_access_between_nodes
+
+# Get the current directory
+base_address=`dirname "$(realpath $0)"`
+# THIS NEEDS TO CHANGE AS PER EXPERIMENT DETAILS
+master_node_login=pmrane@ms1028.utah.cloudlab.us
```

Comment:
as argument

Reply:
This change has been made in the newest commit